### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by jinsu-kim** [dandelin/Dynamic-memory-networks-plus-Pytorch#4]
Hi, 
Thank you for sharing nice codes!!
It will be really helpful to me.

Could you explain how to use pretrained model to get an accuracy?
I am new to pytorch.
